src/components/createAppointment/MemberDetail.tsx

```diff
@@ -0,0 +1,78 @@
+import { Avatar, Chip, Typography, Stack } from "@mui/material";
+import MaleIcon from "@mui/icons-material/Male";
+import FemaleIcon from "@mui/icons-material/Female";
+import TransgenderIcon from "@mui/icons-material/Transgender";
+import CakeIcon from "@mui/icons-material/Cake";
+import { User } from "@/types/User";
+import { Gender } from "enum/gender";
+
+type props = {
+  value: User | null;
+};
+
+const profile_layout = {
+  width: "50vw",
+  minWidth: "200px",
+};
+const stack_design = {
+  width: "400px",
+  height: "50%",
+  boxSizing: "border-box",
+  background: "#FFFFFF",
+  border: "1px solid rgba(0, 0, 0, 0.2)",
+  borderRadius: " 15px",
+};
+
+const avatar = { width: 120, height: 120 };
+
+export default function MemberDetail(props: props) {
+  /*const [text, setText] = useState<string>("");
+  setText(props.value.description);*/
+  return (
+    <>
+      {props.value && (
+        <Stack
```

Comment:
The corners of the box still appear as square.

Reply:
Removed it